.changeset/sixty-seahorses-count.md

```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-react': patch
+---
+
+Fixed an issue where `Select` was being positioned incorrectly after first render.
```

Comment:
To confirm, do we need to say the *dropdownMenu* was positioned incorrectly?

```suggestion
Fixed an issue where `Select`'s dropdownMenu was being positioned incorrectly after its first render.
```

Reply:
how about this?
```suggestion
Fixed an issue where `Select` menu was being positioned incorrectly after first render.
```

Reply:
Sounds good, but I feel with the `'s`, the meaning is a bit clearer. Also, without the `its`, it could be unclear if the "first render" is of the *Select component* or of *its menu*.

```suggestion
Fixed an issue where `Select`'s menu was being positioned incorrectly after its first render.
```

But maybe I'm nitpicking 😅 Your code suggestion is good too.